plugins/outputs/plugin.go

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright 2022 Aspect Build Systems, Inc.
+ *
+ * Licensed under the aspect.build Commercial License (the "License");
+ * you may not use this file except in compliance with the License.
+ * Full License text is in the LICENSE file included in the root of this repository.
+ */
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	goplugin "github.com/hashicorp/go-plugin"
+
+	"aspect.build/cli/pkg/bazel"
+	"aspect.build/cli/pkg/plugin/sdk/v1alpha2/config"
+	aspectplugin "aspect.build/cli/pkg/plugin/sdk/v1alpha2/plugin"
+)
+
+func main() {
+	goplugin.Serve(config.NewConfigFor(New()))
+}
+
+type Outputs struct {
+	aspectplugin.Base
+}
+
+func New() *Outputs {
+	return &Outputs{}
+}
+
+func (plugin *Outputs) CustomCommands() ([]*aspectplugin.Command, error) {
+	return []*aspectplugin.Command{
+		aspectplugin.NewCommand(
+			"outputs",
+			"Inspect declared outputs.",
+			"Queries for the outputs declared by actions generated by the given target.",
+			func(ctx context.Context, args []string, bzl bazel.Bazel) error {
+				if len(args) < 1 {
+					return fmt.Errorf("A label is required as the first argument to aspect outputs")
+				}
+				query := args[0]
+				var mnemonicFilter string
+				if len(args) > 1 {
+					mnemonicFilter = args[1]
+				} else {
+					mnemonicFilter = ""
+				}
```

Comment:
The zero-value of the string in line 46 is already an empty string.
```suggestion
				}
```